recipes/pcapplusplus/all/test_v1_package/test_package.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
```

Comment:
This file can be removed. Re-use directly from test_package folder.

Reply:
Done in https://github.com/conan-io/conan-center-index/pull/14712/commits/4a0959d18aaeb0d995400b3cff6ccea91b38b080